theming.js

```diff
@@ -17,30 +17,40 @@ const Dash = imports.ui.dash;
 const DND = imports.ui.dnd;
 const IconGrid = imports.ui.iconGrid;
 const Main = imports.ui.main;
+const Panel = Main.panel;
```

Comment:
We usually don't do this, but just use Main.panel. I guess it was to keep the code copied from upstram the same. Not a big deal.

Reply:
Fair point, we certainly don't do this anywhere, plus there is no need for `Panel` to be global.

Addressed here [be41bd1f7db367e4f1907106842a8da7b06af352]